### PR TITLE
I think this is a great update to this file.

### PR DESCRIPTION
**Pull request by pipt**

### CODE REVIEW

File: my-wonderful-file.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 This is a great thing.
+
+This is another great thing.
```
Comment (chdask) [inline my-wonderful-file.txt]: some comment

Comment (pipt) [inline my-wonderful-file.txt]: Great idea!

**Comment by pipt** [comment]
@chdask I've made that change you suggested.

**Comment by chdask** [comment]
ok cool let merge!

**Pull request merged**